### DISCUSSION TITLE
Differentiating / distancing this project from qudi (old)

### DISCUSSION THREAD

**Discussion by gemmell** [Ulm-IQO/qudi-core#55] [General]
As an experienced sw engineer I like the architecture & concepts in this project and can see how it's grown out of the old qudi. However, coming in fresh having been told about qudi by others, I am led to the old project which is tied to python 3.6 and didn't quite fulfil its lofty goals. I can see why qudi-core was born and I think there needs to be a greater emphasis that this is the thing that the community should push. It was not clear to me the intent of qudi-core and how it differentiated from the other qudi and in fact it was only by chance that I found qudi-core.

For example perhaps on the readme for qudi direct them to qudi-core and say "Qudi-core is the next evolution of qudi, modern, and more modular to allow greater contribution from the community". Just something that flags to people like me that if I am looking for where to start with qudi, then actually qudi-core is the place I should start. At least in the current readme.md something to the effect that this is the next iteration of qudi? Thoughts?

**Comment by timoML** [comment]
Thanks for pointing that out. We always appreciate 'outside views' on our docs. It was definetly planned to put a deprecation warning into (old) qudi's readme.md. Back then we decided to treat the current qudi-iqo-modules release (v0.4.0) as an unofficial 'testing' release and only put this note after the first offical release of qudi-iqo-modules. However, since then there's been quite some while of testing and I personally would vote for putting this note already now. We'll come to a maintenance team decision on that shortly.

**Reply by timoML** [reply]
The deprecation is now reflected in the [readme](https://github.com/Ulm-IQO/qudi/blob/master/README.md).
